composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "owncloud/files_primary_swift",
+    "require": {
+    },
+    "license": "OCL"
```

Comment:
please add the swift 3rdparty libs in here

Reply:
what are 3rd party here?